### ISSUE TITLE
Australia has over 3,000 recovered cases

### ISSUE THREAD

**Issue by k-dhingra93** [CSSEGISandData/COVID-19#2141]
Cases in Australia have been incorrect for over a week.
Aus government provides daily updates of active cases, deaths and recoveries here:
https://www.health.gov.au/resources/publications/coronavirus-covid-19-at-a-glance

The data for Aus seems to be very inconsistent with official reports

**Comment by MelbourneDeveloper** [comment]
@k-dhingra93 , you're right. Recoveries are being underreported here. The problem is with the [New South Wales record](https://github.com/CSSEGISandData/COVID-19/blob/master/csse_covid_19_data/csse_covid_19_daily_reports/04-11-2020.csv#L2748)

I can see that the states figures are incorrect when I run thios query

```sql
select	Provinces.Name,
		sum(Recoveries) as TotalRecoveries	
from locationdays 
inner join locations
on locations.id = locationdays.locationid
inner join Provinces
on Provinces.id = locations.ProvinceId
inner join Regions
on Regions.id = Provinces.RegionId
where Regions.Name='Australia'
group by Provinces.Name
```

| State                        | Recoveries |
|------------------------------|------------|
| Australian Capital Territory | 59         |
| External territories         | 0          |
| From Diamond Princess        | 0          |
| Jervis Bay Territory         | 0          |
| N/A                          |            |
| New South Wales              | 4          |
| Northern Territory           | 2          |
| Queensland                   | 372        |
| South Australia              | 179        |
| Tasmania                     | 48         |
| Victoria                     | 926        |
| Western Australia            | 216        |
| Switzerland                  | 11623      |
| Portugal                     | 11204      |
| Russia                       | 10881      |
| Israel                       | 7373       |
| India                        | 6711       |

Generated with [this tool](https://github.com/MelbourneDeveloper/COVID-19-DB)

**Comment by shayneoneill** [comment]
The Australian cases appear to be completely missing now. Not sure what the WA recoveries are at now?

**Comment by ChrisJollyAU** [comment]
WA cases and recoveries can be found at https://ww2.health.wa.gov.au/Articles/A_E/Coronavirus/COVID19-statistics

**Comment by BJReplay** [comment]
New south wales wasn't really reporting data until recently.  This data now appears to be up to date.  @k-dhingra93 this looks like it should be OK to close.
